include/openthread/border_agent.h

```diff
@@ -110,6 +110,31 @@ typedef enum otBorderAgentState
     OT_BORDER_AGENT_STATE_ACTIVE  = 2, ///< Border agent is connected with external commissioner.
 } otBorderAgentState;
 
+typedef struct otBorderAgentCounters
+{
+    uint32_t mEpskcActivations;              ///< The number of ePSKc activations
+    uint32_t mEpskcDeactivationClears;       ///< The number of ePSKc deactivations via API
+    uint32_t mEpskcDeactivationTimeouts;     ///< The number of ePSKc deactivations due to timeout
+    uint32_t mEpskcDeactivationMaxAttempts;  ///< The number of ePSKc deactivations due to reached max number of secure
```

Comment:
Would be great if we can fit it in the same line?
```suggestion
    uint32_t mEpskcDeactivationMaxAttempts;  ///< The number of ePSKc deactivations due reached max attemps
```